otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/config/EnvironmentConfig.java

```diff
@@ -130,6 +133,14 @@ private static void applyPropagationFormat(AgentConfig.Builder builder) {
     if (traceReportingType != null) {
       builder.setTraceReporterType(TraceReporterType.valueOf(traceReportingType));
     }
+    String metricReporterAddress = getProperty(REPORTING_METRIC_ENDPOINT);
```

Comment:
should these have default values? With reporting traces, we default to localhost:4317 and OTLP, meaning that for most use-cases users only have to configure the metric endpoint and not the metric type and for simple use cases no configuration is required at all

Reply:
we should default the metrics endpoint to the whatever the traces endpoint is.

Reply:
Yes, these have default values and it is the same as traces. The default type is OTLP, and endpoint is also same as traces endpoint.